index.html

```diff
@@ -44,53 +44,91 @@ <h2>Who</h2>
+      - Vasiliy Burov, <a href="https://twitter.com/vasebur" rel="nofollow">@vasebur</a> (<a href="https://quest.com" rel="nofollow">Quest</a>) &#11088; 🇷🇺 <br>
+      - Mangatas Tondang, <a href="https://twitter.com/tas_kmanager" rel="nofollow">@tas_kmanager</a> (Independent Researcher) &#11088; &#127464;&#127462; <br>
+      - Jaime Flores, <a href="https://twitter.com/remotephone" rel="nofollow">@remotephone</a> (Independent Researcher) &#11088; 🇺🇸 <br>
+      - Ensar Şamil, <a href="https://twitter.com/sblmsrsn" rel="nofollow">@sblmsrsn</a> (Independent Researcher) &#11088; &#127481;&#127479; <br>
+      - Zach Stanford, <a href="https://twitter.com/svch0st" rel="nofollow">@svch0st</a> (<a href="https://twitter.com/CxCyber" rel="nofollow">CyberCX</a>) &#11088; &#127462;&#127482; <br>
+      - Sander Wiebing, (<a href="https://www.nfir.nl/" rel="nofollow">NFIR B.V.</a>) &#11088; &#127475;&#127473; <br>
+      - Jonathan Cheong, (Independent Researcher) &#11088; &#127462;&#127482; <br>
+      - Ömer Günal, <a href="https://twitter.com/ogunal00" rel="nofollow">@ogunal00</a> (Independent Researcher) &#11088; &#127481;&#127479; <br>
+      - Alejandro Ortuno, <a href="https://twitter.com/aomanzanera" rel="nofollow">@aomanzanera</a> (Independent Researcher) &#11088; &#127466;&#127480; <br>
+      - Yuliya Fomina, (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) &#11088; 🇷🇺 <br>
+      - Konstantin Grishchenko, <a href="https://twitter.com/Z3Jpa29z" rel="nofollow">@Z3Jpa29z</a> (Independent Researcher) &#11088; 🇷🇺 <br>
+      - Ivan Dyachkov, (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) &#11088; 🇷🇺 <br>
+      - Nikita Nazarov, <a href="https://twitter.com/NikitaStormwind" rel="nofollow">@NikitaStormwind</a> (<a href="https://www.izsoc.ru/">IZ SOC</a>) &#11088; 🇷🇺 <br>
+      - Natalia Shornikova, (<a href="https://www.izsoc.ru/">IZ SOC</a>) &#11088; 🇷🇺 <br>
+      - Jonhnathan Ribeiro, <a href="https://twitter.com/_w0rk3r" rel="nofollow">@_w0rk3r</a> (Independent Researcher) &#11088; &#127463;&#127479; <br>
       <br><br>
       <h2>Thanks to</h2>
       - <a href="https://2019.hack.lu">hack.lu</a> organizers for a place on conference premises during the Sprint #1<br>
-      - Andrey <a href="https://www.behance.net/PolarLetters">PolarLetters</a> for the logo
+      - Andrey <a href="https://www.behance.net/PolarLetters">PolarLetters</a> for the logo<br>
+      - <a href="https://twitter.com/JohnLaTwC">John Lambert</a> for continuos support<br>
       <br><br>
       <h2>Contacts</h2>
-      - <a href="https://twitter.com/oscd_initiative">Twitter</a><br>
-      - <a href="https://join.slack.com/t/oscd/shared_invite/enQtODc1NjA0MzU0NTQ4LWMyMzdkNTJkZDUzZWZkNDk5M2JlNGY0MWRkMDlmMjg3NjBmZjBiMTg0MGRmMzA1YzI4ZWM3ZjZlMGQ3NWMwYzM">Slack</a> 
+      - <a href="https://twitter.com/oscd_initiative">OSCD Twitter</a><br>
```

Comment:
i believe that there is no need to mention that it is "OSCD" twitter %)

Reply:
Done, thx